### PR TITLE
correct name in service.md

### PR DESCRIPTION
**Pull request by Arhell**
fix issue https://github.com/kubernetes/website/issues/26360

### CODE REVIEW

**Comment by Arhell** [comment]
review page 
https://deploy-preview-26392--kubernetes-io-master-staging.netlify.app/docs/concepts/services-networking/service/
/assign @sftim

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ spec:
       targetPort: 9376
 ```
 
-Because this Service has no selector, the corresponding Endpoint object is not
+Because this Service has no selector, the corresponding Endpoints object is not
 created automatically. You can manually map the Service to the network address and port
 where it's running, by adding an Endpoint object manually:
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/service.md:156]:
```suggestion
where it's running, by adding an Endpoints object manually:
```

**Review by sftim — COMMENTED** [review]
This isn't _wrong_, but you might have missed a fix.
/lgtm

**Comment by sftim** [comment]
/hold
/approve
@Arhell feel free to either accept the suggestion or to unhold. Your call.

File: content/en/docs/concepts/services-networking/service.md (hunk 1 above)
Comment (Arhell) [reply content/en/docs/concepts/services-networking/service.md]: Corrected, thanks!

**Review by sftim — COMMENTED** [review]
/lgtm
/hold cancel

Thanks for the fix

**Pull request merged**